src/test/unit/spec1.js

```diff
@@ -0,0 +1,26 @@
+describe("app", function () {
+
+  var $rootScope;
+      $controller;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected an assignment or function call and instead saw an expression.](https://www.codacy.com/app/triplez/open-event-webapp/file/2533300340/issues/source?bid=3233857&fileBranchId=3284028#l4)